src/components/Game/Game.tsx

```diff
@@ -166,43 +162,27 @@ export const Game = () => {
 
       <div
         className={
-          gameStatus === "inprogress" ? "block-start" : "block-start-hidden "
+          gameStatus == GameStatus.inprogress
+            ? styles.block__visible
+            : styles.block__hidden
         }
       >
-        <Button
-          type="submit"
-          handleClick={onSubmit}
-          disabled={
-            gameStatus === "notstarted" || gameStatus === "finished"
-              ? true
-              : false
-          }
-          className="stay"
-        >
+        <Button type="submit" handleClick={onSubmit}>
           stay
         </Button>
-        <Button
-          type="submit"
-          handleClick={onStopSubmit}
-          disabled={
-            gameStatus === "notstarted" || gameStatus === "finished"
-              ? true
-              : false
-          }
-          className="stay"
-        >
+        <Button type="submit" handleClick={onStopSubmit}>
           hit
         </Button>
 
-        <p className="game__message">your points: {pointsPlayer}</p>
+        <p className={styles.message}>your points: {pointsPlayer}</p>
       </div>
 
       <PlayerHand cards={cardsForPlayer} />
       <div>
-        {gameStatus === "finished" && countDealer > 0 ? (
+        {gameStatus == GameStatus.finished ? (
```

Comment:
==?

Reply:
GameStatus.finished && ( .... )

Reply:
i tried to do like that - it didn't work